sopel/cli/plugins.py

```diff
@@ -536,13 +540,17 @@ def main():
         parser.print_help()
         return
 
-    if action == 'list':
-        return handle_list(options)
-    elif action == 'show':
-        return handle_show(options)
-    elif action == 'configure':
-        return handle_configure(options)
-    elif action == 'disable':
-        return handle_disable(options)
-    elif action == 'enable':
-        return handle_enable(options)
+    try:
+        if action == 'list':
+            return handle_list(options)
+        elif action == 'show':
+            return handle_show(options)
+        elif action == 'configure':
+            return handle_configure(options)
+        elif action == 'disable':
+            return handle_disable(options)
+        elif action == 'enable':
+            return handle_enable(options)
+    except config.ConfigurationNotFound as err:
+        tools.stderr(err)
+        tools.stderr('Use `sopel-config init` to create a new config file.')
```

Comment:
Does this also need to catch `KeyboardInterrupt`? I could see especially `sopel-plugins configure x` running into that, since it starts an interactive wizard.

Reply:
I guess we can do that too.